_maps_2024/01-luming.md

```diff
@@ -2,6 +2,6 @@
 day: 1
 creator: Luming Xu
 title: Where to Savor Yum Cha Around Sun Yat-Sen University?
-thumbnail: /assets/thumbnails/2024/01-luming-points.png
-source: https://github.com/Bluebird-hi/30DayMapChallenge_2024/blob/main/01-Points.R
+thumbnail: /assets/thumbnails/2024/01-Luming-Points.png
+source: https://github.com/Bluebird-hi/30DayMapChallenge_2024/blob/main/01-Points/01-Points.R
```

Comment:
:+1: Good move on correcting the capitalization. Most non-windows operating systems are case-sensitive for file and folder names. I'll have to keep a closer eye out in pull requests.

Reply:
@Bluebird-hi I was wrong! The capitalization of the image file name is lowercased, so this should be too. Files can have uppercase characters, but it's just important that you're consistent when referring to file names.

Reply:
I got it! Thanks for correcting it!